windows_containers/windows-containers-release-notes-5-x.adoc

```diff
@@ -21,13 +21,21 @@ ifndef::openshift-origin[]
 
 {productwinc} is provided and available as an optional, installable component. Windows Container Support for Red Hat OpenShift is not part of the {product-title} subscription. It requires an additional Red Hat subscription and is supported according to the link:https://access.redhat.com/support/offerings/production/soc/[Scope of coverage] and link:https://access.redhat.com/support/offerings/production/sla[Service level agreements].
 
-You must have this separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal]. 
+You must have this separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
 For more information, see the Red Hat OpenShift Container Platform Life Cycle Policy document for link:https://access.redhat.com/support/policy/updates/openshift#windows[Red Hat OpenShift support for Windows Containers].
 
-If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support. 
+If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
 endif::openshift-origin[]
 
+[id="wmco-5-1-0"]
+== Release notes for Red Hat Windows Machine Config Operator 5.1.0
+=== New features and improvements
+[id="wmco-5-1.0-node-certificates"]
+==== Windows node certificates are updated
+
+With this release, the WMCO updates the Windows node certificates when the node certificates are rotated.
```

Comment:
It might sound better the following: "With this release, the WMCO updates the Windows node certificates when the kubelet client CA certificate is rotated."
What do you think @jrvaldes  ?

Reply:
LGTM